register_sync.go

```diff
@@ -0,0 +1,11 @@
+package browser
```

Comment:
Nice catch! Thanks :)

Reply:
Fixed here: https://github.com/grafana/xk6-browser/pull/1373/commits/1c85fa1f4f8ecea58257c4bc703da470fdb73034

I moved the async to `register.go` as it will be the default one. I renamed the other to `sync_register.go`, so we don't forget to remove it.